analysed-packages/atlassian/version-3.41.3/atlassian-3.41.3-OSS-disclosure.txt

```diff
@@ -0,0 +1,226 @@
+PackageName: site-packages_atlassian-with-distinfo.zip
+PackageVerificationCode: 6f727b7368037c4d5dbbdcbda3b058560582c1f9
+PackageChecksum: SHA256: 014f7aa933dbfa7a7d32f1ef5cf51d1470e221aae9530860039436b2729fc7ef
+PackageLicenseDeclared: Apache-2.0

```

Comment:
As already mentioned in my comment on attrs licenses which are cntained in the SPDX license list are not contained in the SPDX files, This you can verfy in the tag value file, it contains no Apache-2.0 license text, it contains only the text of LicenseRef-fossology-Apache-2.0-af5b3d00d6efa4c915c087dc57f6ae02